--- a/subprojects/gstreamer/plugins/elements/gstqueue2.c	2018-03-23 14:33:51.033985808 +0100
+++ b/subprojects/gstreamer/plugins/elements/gstqueue2.c	2018-03-23 14:34:37.526074183 +0100
@@ -115,7 +115,7 @@
 /* default property values */
 #define DEFAULT_MAX_SIZE_BUFFERS   100  /* 100 buffers */
 #define DEFAULT_MAX_SIZE_BYTES     (2 * 1024 * 1024)    /* 2 MB */
-#define DEFAULT_MAX_SIZE_TIME      2 * GST_SECOND       /* 2 seconds */
+#define DEFAULT_MAX_SIZE_TIME      0 * GST_SECOND       /* deactivated */
 #define DEFAULT_USE_BUFFERING      FALSE
 #define DEFAULT_USE_TAGS_BITRATE   FALSE
 #define DEFAULT_USE_RATE_ESTIMATE  TRUE
